### PR TITLE
new icon: numpy (original, original-wordmark)

### PR DESCRIPTION
**Pull request by kimjiwook0129**
new icon: numpy (original, original-wordmark)

I've re-made the PR and made it from the develop branch.

I know this must be such simple task to do, but took me some time to learn some git tools and understand them since it is my first time trying to contribute to an open-source. But thanks for waiting me and for letting me to contribute!

Please feel free to tell me if anything still needs to be worked on.

Thanks for helping me out!

### CODE REVIEW

**Comment by theblobscp** [comment]
Looks good, but I'm a little confused. On the xml declaration the version is 1.0. But on the svg tag it says the version is 1.1. Any reason for that?

**Comment by kimjiwook0129** [comment]
Hmm... Not too sure about the reason behind, 
I think one of the versions should have been created while croping and the other during strokes-to-path removal (I did these 2 steps separately).
Shall I make them consistent?
```diff
- <?xml version="1.0" encoding="UTF-8"?><svg width="128pt" height="128pt" version="1.1" ...
+ <?xml version="1.1" encoding="UTF-8"?><svg width="128pt" height="128pt" version="1.1" ...
```
OR I can just delete the attribute if they are unnecessary:
```diff
- <?xml version="1.0" encoding="UTF-8"?><svg width="128pt" height="128pt" version="1.1" ...
+ <?xml encoding="UTF-8"?><svg width="128pt" height="128pt" ...
```

**Comment by theblobscp** [comment]
Uh I'm not a team member, I'm just some guy with experience with failing miserably at svgs. But to answer your question. You can remove the declaration (most svgs don't have it) and keep the version attribute

**Comment by kimjiwook0129** [comment]
Declarations removed like other svgs, thanks!

File: devicon.json [hunk 1]
```diff
@@ -2885,6 +2885,24 @@
...
+            "library",
+            "python"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark"
+            ],
+            "font": [
+                "original-wordmark"
```
Comment (Panquesito7) [inline devicon.json:2900]:
```suggestion
                "original",
                "original-wordmark"
```

File: devicon.json [hunk 2]
```diff
@@ -2885,6 +2885,24 @@
...
+            ]
+        },
+        "color": "#4DABCF",
+        "aliases": []
```
Comment (Panquesito7) [inline devicon.json:2904]:
```suggestion
        "aliases": [
            {
                "base": "original",
                "alias": "plain"
            }
        ]
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Good work! Almost there. 😄

**Comment by kimjiwook0129** [comment]
Edited!

**Review by Panquesito7 — APPROVED** [review]
LGTM. Thank you for your contribution! 😄👍🎉

**Comment by Thomas-Boi** [comment]
@kimjiwook0129 the icons look great and ready to be merge. However, I'm almost done with a peek-bot upgrade that I want to test on this PR. Pretty much, Icomoon has issues with strokes in SVGs. Yours don't have them BUT they do use `stroke=none`. I just want to test whether Icomoon would have an issue with this.

Give me a week to get this done. After that, I'll merge this PR in whether the testing works since in theory, your SVGs are good to go.

**Comment by kimjiwook0129** [comment]
> Panquesito7 

Thank you!

> Thomas-Boi 

Please take your time, and thank you!

**Review by Thomas-Boi — APPROVED** [review]
Hi @kimjiwook0129,

I'm going to merge your PR into `develop`. The feature I'm working requires the next release but I want your icons in that release as well. Thus, I've tested your icon manually to support my feature and all is well 😄. Thank you for your icons and we appreciate your work 👍

**Review by Panquesito7 — APPROVED** [review]

**Pull request merged**